### DISCUSSION TITLE
higress 业务api接口限流配置

### DISCUSSION THREAD

**Discussion by liuflylove666** [higress-group/higress#1222] [Announcements]
我这边有个场景需要限制业务api接口qps，请问如何实现？类似云上配置

![image](https://github.com/user-attachments/assets/72870bdd-d35c-466e-afb2-9552c25bc67d)

**Comment by johnlanni** [comment]
单机限流：https://higress.io/docs/latest/user/annotation-use-case/#%E5%8D%95%E6%9C%BA%E9%99%90%E6%B5%81

集群限流：https://higress.io/docs/latest/plugins/traffic/cluster-key-rate-limit/

**Comment by liuflylove666** [comment]
我看过这个都是基于key的限制，没有像云上一样直接在页面上操作，针对api接口做QPS限制操作，我看都是基于key才行。

**Reply by johnlanni** [reply]
https://higress.io/docs/latest/user/annotation-use-case/#%E5%8D%95%E6%9C%BA%E9%99%90%E6%B5%81

这个就是针对 API 接口的
